relationships/synthesis/NR_INFRA-KUBERNETES_CONTAINER-to-EXT-SERVICE.yml

```
@@ -0,0 +1,38 @@
+relationships:
+  - name: nrInfraK8sContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: entity.type
+        anyOf: [ "SERVICE" ]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        buildGuid:
+          account:
+            lookup: true
+          domain:
+            value: INFRA
+          type:
+            value: CONTAINER
+            valueInGuid: NA
+          identifier:
+            fragments:
+              - value: "k8s:"
+              - attribute: k8s.cluster.name
+              - value: ":"
+              - attribute: k8s.namespace.name
+              - value: ":"
+              - attribute: k8s.pod.name
+              - value: ":container:"
+              - attribute: k8s.container.name
+            hashAlgorithm: FARM_HASH
+      target:
+        extractGuid:
+          attribute: entity.guid
+          entityType:
+            value: entity.type
```

You can set the entity type directly as `SERVICE` using

```suggestion
            value: "SERVICE"
```

and delete the attribute `entity.type` under `conditions`

---

@mborroni, Thanks for reviewing this PR. I have updated the PR as per your suggestion.